lib/tasks/migrations.rake

```diff
@@ -9,8 +9,14 @@ namespace :migrations do
     require File.join(File.dirname(__FILE__), '..', '..', 'config', 'environment')
     puts AppConfig.environment.s3.key
 
-    connection = Aws::S3.new( AppConfig.environment.s3.key, AppConfig.environment.s3.secret)
-    bucket = connection.bucket(AppConfig.environment.s3.bucket)
+    connection = Fog::Storage.new({
+      provider:              AppConfig.environment.s3.provider.get,
+      aws_access_key_id:     AppConfig.environment.s3.key.get,
+      aws_secret_access_key: AppConfig.environment.s3.secret.get,
+      region:                AppConfig.environment.s3.region.get
+    })
```

Comment:
Layout/FirstHashElementIndentation: Indent the right brace the same as the first position after the preceding left parenthesis.